### PR TITLE
Callback leveraging livelossplot

### PR DESCRIPTION
**Pull request by ottonemo**
Live plotting in Jupyter Notebook.

This is a WIP but comments are welcome.

Uses https://github.com/stared/livelossplot/ to provide this feature.

### CODE REVIEW

**Comment by benjamin-work** [comment]
Is there any meaningful way to add a test to this feature? Or maybe add an example to a notebook?

**Comment by benjamin-work** [comment]
I tried this:

```
# in Advanced_Usage.ipynb
net = NeuralNetClassifier(
    ClassifierModule,
    max_epochs=20,
    lr=0.1,
    callbacks=[
        ('plot', LivePlot()),
    ],
)
net.fit(X, y)
```

Interestingly, it only plotted the training loss, even though a valid loss exists. Furthermore, the output from `PrintLog` is cleared by the plot, but I guess we have to accept that.

`LivePlot` also had a 3x overhead for this specific example. This is not too relevant, but one might think about a feature to only update the plot every N epochs.

**Comment by thomasjpfan** [comment]
If there is still interest in this, there is another similar library that may be useful: https://github.com/fastai/fastprogress

**Comment by ottonemo** [comment]
I don't see that `fastprogress` does more than what we support now. The plotting is independent from the progress bar as far as I am concerned. I'd still be interested in something like `livelossplot` but I think that it would be better to have a better integration with plotting tools like visdom (for example).

Since this PR is open for quite a while I think we should abandon this feature in favor of better support for visdom or use a more powerful library such as https://github.com/waleedka/hiddenlayer.
